hikari/__init__.py

```diff
@@ -27,12 +27,8 @@
 the REST API.
 """
 
-# We need these imported explicitly for the __all__ to be visible due to
-# Python's weird import visibility system.
-from hikari import config
-from hikari import errors
-from hikari import events
-from hikari import models
+from __future__ import annotations
```

Comment:
Do we really need these?

Reply:
Consistency, and reduces import time.

There is no reason not to include them.